pkg/image/extensions.go

```diff
@@ -0,0 +1,94 @@
+package image
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/Masterminds/semver/v3"
+)
+
+type Extensions struct {
+	Config         ExportConfig
```

Comment:
I do not see any usage of this field. We can consider dropping it unless there is something I do not realize.